Documentation/source/Updating to 3.2.0.md

```diff
@@ -78,6 +78,10 @@ if hasPayFive {
 }
 ```
 
+### Custom font
+Now is possible to specify a custom font for the whole library, being not needed to specify a font for each text.
```

Comment:
Now **it** is possible to specify a custom font for the whole library. Specifying a font for each text is not necessary.
 +You just need to specify the `GiniVisionFont` in the `GiniConfiguration.font` property. Previous**ly** defined fonts are no longer needed